digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala

```diff
@@ -194,17 +194,53 @@ class VerificationDao {
     id
   }
 
-  def updateAssetTypeVerification(municipalityId: Int, assetTypeId: Int, lastUserModification: Option[String], lastDateModification: Option[DateTime], numberOfAsset: Int, refreshDate: Option[DateTime], suggestedAssets: String): Unit = {
-    sqlu"""update municipality_verification
-           set last_user_modification = $lastUserModification,
-               last_date_modification = $lastDateModification,
-               number_of_assets = $numberOfAsset,
-               refresh_date = $refreshDate,
-               suggested_assets = $suggestedAssets
-         where municipality_id = $municipalityId
-           and asset_type_id = $assetTypeId
-           and valid_to is null
-      """.execute
+  def insertAssetTypeVerificationQuery(municipalityCode: Int, assetTypeId: Int, verifiedBy: Option[String],
+                                  modifiedBy: Option[String], modifiedDate: Option[DateTime],
+                                  numberOfAssets: Int, refreshDate: Option[DateTime], suggestedAssets: String): String = {
+    val (verifiedByUser, verifiedDate) = if (verifiedBy.nonEmpty) (verifiedBy.get, "current_timestamp") else ("null", "null")
+    val lastUserModification = if (modifiedBy.nonEmpty) s"'${modifiedBy.get}'" else "null"
+    val lastDateModification = if (modifiedDate.nonEmpty) s"'${modifiedDate.get}'" else "null"
```

Comment:
eikös null tuota tyhjän syötön kantaan. Toinen mitä mietityttää syöttääkö tää nyt merkkijonon "null"

Reply:
https://www.postgresql.org/message-id/Pine.LNX.4.33.0402181239030.2832-100000%40css120.ihs.com
https://www.enterprisedb.com/postgres-tutorials/how-null-and-empty-strings-are-treated-postgresql-vs-oracle

Reply:
Tää syöttää nyt null arvon kantaan, jos meillä ei ole arvoa tiedossa. Testauksien perusteella ainakin tuotetaan samanlaista dataa mitä ennen näitä muutoksia.

Reply:
ok